auth/kerberos/src/utf16_decode/build-using-csc.sh

```diff
@@ -16,11 +16,6 @@ fi
 echo "dotnethome=$dotnethome"
 
 dotnetlib=$dotnethome/shared/Microsoft.NETCore.App/$fwkver
-if [ -d /usr/share/dotnet/packs/Microsoft.NETCore.App.Ref/$fwkver/ref/net8.0/ ]; then
```

Comment:
Why was this removed ?

Reply:
we don't need conditional logic anymore since the project is updated to net8.0. NativeAot is supported for versions >net7.0

Reply:
NativeAot was added after discussion, I suppose

Reply:
Yes, that's correct